src/v2/view-builder/views/idp/AuthenticatorIdPEnrollView.js

```diff
@@ -1,6 +1,5 @@
 import { loc } from 'okta';
-import BaseIdPAuthenticatorBody from './BaseIdPAuthenticatorBody';
-import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
```

Comment:
TO confirm the BaseAuthenticatorView.js is not deleted ? Assume its used somewhere else too

Reply:
yeah, we are using this and creating another Base class in BaseIdpAuthenticar.js and using that here.